tests/Tests/400__ods_dashboard/adminUITest.robot

```diff
@@ -0,0 +1,159 @@
+*** Settings ***
+Library         OpenShiftCLI
+Library         OpenShiftLibrary
+Resource        ../../Resources/Common.robot
+Resource        ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+
+
+*** Test Cases ***
+Verify If Unauthorisez User Can Not Change The Permission
```

Comment:
maybe this TC could be split into 2 or more, since it's not just checking for unauthorized users which is only the last portion of it

Reply:
i know this TC looks big but we cannot  split It as its a senerio